packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: For Developers
```

Comment:
Should page titles be sentence or title case?

Reply:
Changed. Any alternative ideas for what this title should be?